### PR TITLE
clarified input defaulting description

### LINKED ISSUE #16: Lesson 5 - New Task - unclear explanation
Current text:
*First note that the inputs name is not mapped to a value that is being passed in, as it is in the print_start task. It's not necessary here because the input name in the operation is the same as the input name in the flow. The value input to the flow will get passed to the operation input with the same name.*

I would rephrase it to something like:
*First note that for the check_address task, the input name is not given a default value, as it is in the print_start task. It's not necessary here because the input name in the check_availability operation used here is the same as the input name in the flow. The value input to the flow will get passed to the operation input with the same name.*

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Fixes #16

### CODE REVIEW

**Comment by oritstone** [comment]
@SamMarkowitz you need to merge it now to the new repo, right?

**Comment by SamMarkowitz** [comment]
It was already in the new one.
